index.html

```diff
@@ -12,12 +12,12 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&family=Roboto:wght@700&display=swap" rel="stylesheet">
 <body>
     <nav class="toolbar" id="toolbar">
-        <p class="logo">My Logo</p>
+        <a class="logo">My Logo</a>
         <div class="menu">
             <button class="dropdown-button"><span class="material-symbols-outlined">
                 menu
                 </span></button>
-            <div class="dropdown-menu">
+            <ul class="dropdown-menu">
                 <button href="#">Portfolio</button>
```

Comment:
[OPTIONAL] I think it would be better if you use `<a>` tag for navigation. Button tag is mostly used for opening popup, deleting a task from the to-do list etc. If you are leading the user to a URL, it would be better to use `<a>`tag with its href value 👍